eng/ci/templates/official/jobs/pack-host-artifacts.yml

```diff
@@ -21,7 +21,13 @@ jobs:
     displayName: Download prelaunch artifacts
     inputs:
       artifactName: _preLaunchAppPackages
-      path: $(Build.SourcesDirectory)/host/dist/portable
+      path: $(Build.SourcesDirectory)/host/dist/portable/prelaunchapps
+
+  - task: DownloadPipelineArtifact@2
```

Comment:
Are we supposed to use `DownloadPipelineArtifact`? 1ES has its own syntax for artifact inputs:

https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/features/inputs/pipeline-artifact